internal/cmd/pipeline/command_update.go

```diff
@@ -60,17 +70,19 @@ func (c *command) update(cmd *cobra.Command, args []string) error {
 	description, _ := cmd.Flags().GetString("description")
 	sqlFile, _ := cmd.Flags().GetString("sql-file")
 	secrets, _ := cmd.Flags().GetStringArray("secret")
+	ksqlCluster, _ := cmd.Flags().GetString("ksql-cluster")
```

Comment:
why are these err all being ignored？

Reply:
We have empty check later. What do you suggest here?

Reply:
```suggestion
	ksqlCluster, err := cmd.Flags().GetString("ksql-cluster")
	if err != nil {
		return err
	}
```

Reply:
Yes please!